extensions/sql-migration/src/api/utils.ts

```diff
@@ -780,7 +780,7 @@ export async function createPowershellscriptContentContainer(view: azdata.ModelV
 		},
 	}).component();
 
-	const path1 = path.join(__dirname, '../../scripts/SHIR-auto-configuration.ps1');
+	const path1 = path.join(__dirname, '../scripts/SHIR-auto-configuration.ps1');
```

Comment:
That is why i had asked this question in previous fix PR:  https://github.com/microsoft/azuredatastudio/pull/25449/files#r1507266817

so it seems root path is: extensions\microsoft.sql-migration-1.5.4\images